src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -845,8 +962,13 @@ private ArgumentSignatureNode ParseArgumentSignature(MethodBase method)
                 argumentSignature.AddArgument(paramNode);
             }
 
-            argumentSignature.IsVarArg = parameters.Any()
-                && parameters.Last().GetCustomAttributes(typeof(ParamArrayAttribute), false).Any();
+            var t = parameters.Any();
```

Comment:
please don't use variable names like this - it makes searching for them in a regular text editor or github ui impossible and makes understanding intent just as hard.

Is this used somewhere?

Reply:
Nope sorry about that, had to do a lot of debugging when trying to figure out how all of this was working and some of these i forgot to remove again.